cloud/pkg/cloudhub/servers/server.go

```diff
@@ -37,13 +37,22 @@ func StartCloudHub(config *util.Config, messageq *channelq.ChannelMessageQueue)
 	handler.InitHandler(config, messageq)
 
 	svc := server.Server{
-		Type:       api.ProtocolTypeQuic,
+		Type:       protocolType,
 		Addr:       fmt.Sprintf("%s:%d", config.Address, config.QuicPort),
```

Comment:
The `QuicPort` also should move to switch :) @du2016

Reply:
sorry, i'll change it